From 39b28692ba4592acb4e3506c22cad9df3ccd80ff Mon Sep 17 00:00:00 2001
From: 221220000-Zhang San <zhangsan@foo.com>
Date: Wed, 1 Oct 2025 17:20:38 +0800
Subject: [PATCH 29/30] submit patch

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 8c04b3b..cdf8de1 100644
--- a/.gitignore
+++ b/.gitignore
@@ -13,3 +13,4 @@
 /nvboard
 /am-kernels
 /fceux-am
+!/patch/*
\ No newline at end of file
-- 
2.34.1

